command/oauth/cmd.go

```diff
@@ -181,6 +181,10 @@ $ step oauth --client-id my-client-id --client-secret my-client-secret \
 				Name:  "scope",
 				Usage: "OAuth scopes",
 			},
+			cli.StringFlag{
+				Name:  "prompt",
+				Usage: "Whether the Authorization Server prompts the End-User for reauthentication and consent",
```

Comment:
@Slamdunk I think we should add to the flag description what are the defined values, either just by name or even with a description formatting those like
https://github.com/smallstep/cli/blob/533e53e9ef691bc5c52cd548355b9b2c06eddaf1/command/crypto/jwk/create.go#L202-L215

Reply:
Done:

![image](https://user-images.githubusercontent.com/152236/107476275-cc6d6880-6b75-11eb-9067-f684736e17c1.png)